sound/soc/sof/ipc4-topology.c

```diff
@@ -1033,8 +1033,13 @@ static int sof_ipc4_init_audio_fmt(struct snd_sof_dev *sdev,
 				   struct sof_ipc4_base_module_cfg *base_config,
 				   struct snd_pcm_hw_params *params,
 				   struct sof_ipc4_available_audio_format *available_fmt,
-				   struct sof_ipc4_pin_format *pin_fmts, u32 pin_fmts_size)
+				   struct sof_ipc4_pin_format *pin_fmts, u32 pin_fmts_size,
```

Comment:
commit message: 
"in order to" repeated

"the topology will have to indicate the pin binding" 
Doesn't this do this already?

"and the pin formats per output pin for the module-copier.N.1
widget"

Could we instead use the input pin format for efx2?

Reply:
Fixed both commit messages now to say that the first patch fixes PCM 1 start and the second patch is to fix the simultaenous use-case